tex/pgfcirc.defines.tex

```diff
@@ -477,10 +537,8 @@
             \pgf@circ@res@down = \pgf@y
             \pgf@circ@scaled@Rlen=\scaledRlen
             \pgfstartlinewidth=\pgflinewidth
-            % no rounded corners nor default arrows on bipole's bodies
-            \pgf@circ@reset@arrows@rounded
+            % we used to reset arced corners here --- now it's in the hooks. I hope.
             #7
```

Comment:
The newline after `#7` is not ignored and will be converted to a space. And when `#7` itself ends with a space (usually converted from newline), we'll have two consecutive space chars in some node shape's background path. It won't affect any usual use, but prevents one from doing patches, as I complained in code comment in https://github.com/circuitikz/circuitikz/issues/605#issuecomment-1030745366. 

`#7%` would resolve the issue.

Reply:
I hope I have done it (and quite a bunch more around...). Thanks a lot!